electron-builder.yml

```diff
@@ -4,7 +4,7 @@ directories:
   output: releases
   buildResources: app-icons # this is where app-icons is store
 appId: com.microsoft.vott
-artifactName: '${productName}-${os}.${ext}'
+artifactName: '${productName}-${version}-${platform}.${ext}'
```

Comment:
where is `platform` coming from:

Reply:
it's a standard electron macro: `platform “darwin” | “linux” | “win32” | “mas”`

https://www.electron.build/configuration/configuration#configuration